### PR TITLE
Fixes #426 by adding My Groups route

### LINKED ISSUE #426: My groups
Provide a "my groups" user route to list all the groups the user is a member.

### PR DESCRIPTION
**Pull request by heddn**

### CODE REVIEW

**Review by MPParsley — APPROVED** [review]
Not sure if we need the word 'yet' but this is an improvement, thanks

**Comment by heddn** [comment]
This just takes the existing default wording of 'There are no @label yet.' and replaces it with the less technical name of 'memberships'. Previously it was 'og memberships' for the label. Which exposes the implementation too much for my liking.

File: og.routing.yml [hunk 1]
```diff
@@ -43,3 +43,16 @@ og.remove_multiple_roles_confirm:
...
+og.user_memberships:
+  path: '/user/{user}/groups'
+  defaults:
+    _entity_list: 'og_membership'
+    _title: 'My groups'
+  requirements:
+    user: '\d+'
+    _entity_access: 'user.view'
+  options:
+    parameters:
```
Comment (MPParsley) [inline og.routing.yml:56]:
```suggestion
    _admin_route: 'TRUE'
    parameters:
```

I suggest we add this as an admin route so it's themed using the admin theme.

groups
groups admin theme

**Review by MPParsley — APPROVED** [review]

**Comment by MPParsley** [comment]
Closing in favor of #466.

**Pull request closed without merging**